_posts/en/newsletters/2023-01-04-newsletter.md

```diff
@@ -0,0 +1,200 @@
+      Towns's email also includes additional information that will be
+      useful to anyone participating in the signet tests.
+
+    - *Full-RBF peering node:* Peter Todd [announced][todd rbf node] a
+      patch on top of Bitcoin Core 24.0.1 that sets a full-RBF service
+      bit when it advertises its network address to other nodes,
+      although only if the node is configured with `mempoolfullrbf`
+      enabled.  Nodes running the patch also connect to up to four
+      additional peers which had advertised that they support full-RBF.
+      Peter Todd notes that [Bitcoin Knots][], another full node, also
+      advertises the service bit, although it doesn't contain code to
+      specifically peer with nodes advertising full-RBF support.
+
+- **Continued RBF discussion:** in ongoing discussion about enabling
+  [full-RBF][topic rbf] on mainnet, several parallel discussions were
+  held last month on the mailing list:
+
+    - *Full-RBF nodes:* Peter Todd probed full nodes which advertised
+      that they were running Bitcoin Core 24.x and were accepting
+      incoming connections on an IPv4 address.  He [found][todd probe]
+      that about 17% relayed a full-RBF replacement: a transaction which
+      replaced a transaction that did not contain the [BIP125][]
+      nSequence signal.  This indicates those nodes were running with
+      the `mempoolfullrbf` configuration option set to `true`, even
+      though the option defaults to `false`.
+
+    - *Reconsideration of RBF-FSS:*  Daniel Lipshitz [posted][lipshitz
+      fss] to the Bitcoin-Dev mailing list an idea for a type of
+      transaction replacement called First Seen Safe (FSS) where the
+      replacement would pay the original outputs at least the same
+      amounts as the original transaction, ensuring the replacement
+      mechanism couldn't be used to steal from the receiver of the
+      original transaction.  Yuval Kogman [replied][kogman fss] with a
+      link to an [earlier version][rbf-fss] of the same idea posted in
+      2015 Peter Todd.  In a [subsequent][todd fss] reply, Todd
+      described several ways in which the idea is much less preferable
+      than opt-in or full RBF.
+
+    - *Full-RBF motivation:* Anthony Towns [replied][towns rbfm] to a
+      thread about the motivation for various groups to perform
+      full-RBF.  Towns analyzes what economic rationality means---and
+      does not mean---in the context of miner transaction selection.
+      Miners optimizing for very short term profits would naturally
+      prefer full-RBF.  However, Towns notes that miners who have made a
+      long-term capital investment in mining equipment might instead
+      prefer to optimize fee income over multiple blocks, and that might
+      not always favor full-RBF.  He suggests three possible scenarios
+      for consideration.
+
+        Towns also analyzes the phase, "Bitcoin is money for enemies"
```

Comment:
I dont feel strongly on it, but this paragraph seems out of place in a typical Optech newsletter.